### PR TITLE
PR1 - Boiler plate code

### PR DESCRIPTION
**Pull request by iulianpascalau**
- added boilerplate code in flags.go & main.go
- added config definition files
- added workflows, Makefile and changed .gitignore file

### CODE REVIEW

File: .github/workflows/pr-tests.yml [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+        run: |
+          go get -v -t -d ./...
+      - name: Unit tests
+        run: make test
+
+      - name: Upload coverage reports to Codecov
+        uses: codecov/codecov-action@v4.0.1
+        with:
+          token: ${{ secrets.CODECOV_TOKEN }}
+          slug: iulianpascalau/mvx-keys-monitor
```
Comment (andreibancioiu) [inline .github/workflows/pr-tests.yml:33]: Should be updated.

File: go.mod [hunk 2]
```diff
@@ -0,0 +1,27 @@
+module multiversx/mvx-keys-monitor
+
+go 1.20
+
+require (
+	github.com/BurntSushi/toml v0.3.1
```
Comment (andreibancioiu) [inline go.mod:6]: Seems to be a different `toml` library than the one used by `mx-chain-go`. It seems to be fine (many stars, many forks).

Comment (iulianpascalau) [reply go.mod:6]: autocompleted pe goland. Will switch to what we are using. Thanks for the heads-up. :+1:

Comment (iulianpascalau) [reply go.mod:6]: removed

File: .github/workflows/pr-tests.yml (hunk 1 above)
Comment (iulianpascalau) [reply .github/workflows/pr-tests.yml:33]: fixed, thanks

**Review by andreibancioiu — APPROVED** [review]

**Review by miiu96 — APPROVED** [review]

**Pull request merged**